metadata.json

```diff
@@ -56,7 +60,20 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "7"
+        "7",
+        "8"
```

Comment:
do you know if this adds EL9 support as well?

Reply:
The changes were made in such a way to make EL9 support easier.  I figured once this merged will then open new PR to validate EL9.